### PR TITLE
Minor improvements to MS Transferor; don.t query gwmsmon

### LINKED ISSUE #9223: ReqMgr2MS is not getting assigned work from the correct ReqMgr2 endpoint
Testbed reqmgr2ms should be calling the cmsweb-testbed reqmgr2 service, not the production one.
Similar comment goes to reqmgr2ms in cmsweb production, it then has to contact the production reqmgr2 instance.

Error in the log says it all
```
WARNING:reqmgr2ms:UnifiedTransferorManager:The cachefile /data/srv/state/reqmgr2ms/.wmcore_cache_1000058/requests/cmsweb.cern.ch:8443/15744094587142094_GET_request?status=assigned&detail=True does not exist and the service at https://cmsweb.cern.ch:8443/reqmgr2/data/request?status=assigned&detail=True is unavailable - it returned 403 because Forbidden with result: <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN" "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
```

### PR DESCRIPTION
**Pull request by amaltaro**
Fixes #9223 

#### Status
not-tested

#### Description
Minor improvements to MS Transferor module, like:
* read the reqmgr2 and dbs url from the config file
* commented out any queries to gwmsmon, which is failing BIG time at the moment
* log the whole exception/traceback in case of general exceptions

#### Is it backward compatible (if not, which system it affects?)
YES

#### Related PRs
Related to #9216 and #9140 

#### External dependencies / deployment changes
Yes, it needs these deployment changes: https://github.com/dmwm/deployment/pull/776

### CODE REVIEW

File: src/python/WMCore/MicroService/Unified/Transferor.py [hunk 1]
```diff
@@ -110,17 +107,19 @@ def transferor(self, reqStatus='assigned'):
         For references see
         https://github.com/dmwm/WMCore/wiki/ReqMgr2-MicroService-Transferor
         """
+        requestRecords = []
         try:
             # get requests from ReqMgr2 data-service for given statue
             requestSpecs = self.svc.reqmgr.getRequestByStatus([reqStatus], detail=True)
-            requestRecords = [requestRecord(r, reqStatus) for r in requestSpecs]
+            if requestSpecs:
+                for _wf, wfData in requestSpecs[0].items():
```
Comment (vkuznet) [inline src/python/WMCore/MicroService/Unified/Transferor.py]: Replace `_wf`with `_` and it will fix pylint degradation.

**Review by vkuznet — COMMENTED** [review]
Alan, changes are fine except one minor place which you can easily fix and pylint will be happy.

**Comment by amaltaro** [comment]
Pylint issue fixed. I'm merging it now such that I can hand a new tag to Lina asap.

**Comment by amaltaro** [comment]
Now that I looked at this code, I'm very concerned about its maintainability, it relies on so many source of information, that I can't see it working without often expert intervention. We gotta make it simpler and robust!

Having said that, I think we need to get started with a few other things, and only then jump straight into a code cleanup, otherwise it just makes it much harder to work on.

**Comment by vkuznet** [comment]
Alan, please be specific. Your messages are too generic and it is hard to understand what's wrong. Which code are you talking about, the unified part, the transferor part, the monit part?

**Pull request merged**